test/dummy/config/application.rb

```diff
@@ -7,5 +7,6 @@
 
 module Dummy
   class Application < Rails::Application
+    config.load_defaults Rails.version[/^\d+\.\d+/]
```

Comment:
Ah OK, so this needs to be dynamic so that we pull in the defaults for the rails version we're using in the part of the Travis matrix which is being run at the moment? 

Seems legit 👍

Reply:
Exactly!